### ISSUE TITLE
How to keep a cloned repo up to date?

### ISSUE THREAD

**Issue by martinholmer** [open-source-economics/Tax-Calculator#1772]
In pull request #1711, @derrickchoe said:
> Thank you all for your patience-- this relatively simple addition to tax-calculator was a great introduction to git usage. As @hdoupe mentioned to me, one of the potential issues with my master branch was my failure to use "git push" to update my remote repository continually after updating my local master branch. I'm still learning how my local repository, "derrickchoe/tax-calculator.git" remote repository, and the central tax-calculator repository interact with one another. I'm looking forward to making smoother updates to tax-calculator in the future.

So, @derrickchoe and @hdoupe think the problem in #1711 was that a new user, who was following the Workflow instructions in the Contributor Guide, let his mater branch get 379 commits behind the central Tax-Calculator master branch because he did not periodically execute `git push origin master`.
And once he did, everything was fine with pull request #1711.

So, it seems to me that a change @MattHJensen made in the Contributor Guide some time ago needs to be reversed.  The sync instructions used to say you needed to do three git commands:
```
git fetch upstream
git merge upstream/master
git push origin master
```
But @MattHJensen changed the sync instructions to be just two git commands:
```
git fetch upstream
git merge upstream/master
```
Does the experience of @derrickchoe imply that we need to move back to the three commands to sync cloned repositories?  Or have @derrickchoe and @hdoupe misdiagnosed the problems @derrickchoe was having when preparing pull request #1711?

**Comment by hdoupe** [comment]
Doing `git push origin master` is necessary to keep the remote fork up to date.  That appeared to be the main part of the problem in #1711.  I think it should be added back to the Contributor Guide.

**Comment by martinholmer** [comment]
Issue #1772 has been resolve by the RTD changes in merged pull request #1773.
